composer.json

```diff
@@ -10,6 +10,7 @@
     ],
     "require": {
         "php": "^7.4 || ^8",
+        "doctrine/dbal": "^2 || ^3",
```

Comment:
Hier dan iig even de minimale versie van v2 van maken waar `executeQuery` beschikbaar is?

Reply:
Dat lijkt https://github.com/doctrine/dbal/blob/2.13.1/lib/Doctrine/DBAL/Statement.php te zijn. Het gaat hier om het aanroepen van `executeQuery` op een `Statement` in `src/Doctrine/NativeQueryPagable.php`